source/CIGameSettings.h

```diff
@@ -2,15 +2,20 @@
 //  CIGameSettings.h
 //  CoreImpact
 //
-// This model class stores game settings that users can change from the menu scene
+//  This model class stores game settings that users can change from the menu scene
 //
 //  Created by William Long on 4/15/21.
 //  Copyright © 2021 Game Design Initiative at Cornell. All rights reserved.
 //
 
 #ifndef __CI_GAME_SETTINGS_H__
 #define __CI_GAME_SETTINGS_H__
+#include <cugl/cugl.h>
+#include "CIGameConstants.h"
 
+/** 
```

Comment:
NIT: not a huge deal, but we don't need comments here when the file is commented at the top